wiki/mkdocs.yml

```diff
@@ -92,14 +92,18 @@ nav:
       - Permissions System: middlewares/permissions.md
       - Cooldown System: middlewares/cooldown.md
   - Dependency Injection: di.md
-  - Localization: localization.md
-  - Extensions:
-      - Overview: extension/overview.md
-      - Writing Extensions: extension/writing.md
-      - Configuration: extension/configuring.md
+  - Message:
+      - Overview: message/overview.md
+      - Emojis: message/emojis.md
+      - Placeholders: message/placeholder.md
+      - Localization: message/localization.md
   - Misc:
       - Embed API: misc/embeds.md
```

Comment:
maybe move embed api to message?

Reply:
because embeds are still messages

Reply:
Moving embeds to message seems right to me